Pipfile

```
@@ -8,15 +8,17 @@ name = "pypi"
 url = "https://pypi.org/simple"
 verify_ssl = true
 
+[packages]
+ansible = "~=8.2.0"
+ansible-core = "==2.15.2"
+
 [dev-packages]
-yamllint = "1.28.0"
-ansible-lint = "6.8.6"
-molecule = {extras = ["docker"], version = "4.0.3"}
-reuse = "1.0.0"
+yamllint = "~=1.32.0"
+ansible-lint = "6.17.2"
+molecule = "~=5.1.0"
```

There is new molecule major release available. Maybe we want to try it?
```suggestion
molecule = "~=6.0.1"
```